src/components/Home/HomeSection.jsx

```diff
@@ -6,8 +6,8 @@ import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 /**
  * @typedef {object} HomeSectionParams
- * @property {string} src - image src
- * @property {string} alt - image alt
+ * @property {string} componentImageSrc - image src
+ * @property {string} componentImageAlt - image alt
```

Comment:
@leekahung mentioned that `alt` should be more aptly named since it's only for the component image. Since `src` was the same, I thought I should lump it in.